jip/maven.py

```diff
@@ -110,6 +110,9 @@ def get_element_tree(self):
             ## we use this dirty method to remove namesapce attribute so that elementtree will use default empty namespace
             pom_string = re.sub(r"<project(.|\s)*?>", '<project>', self.pom_string, 1)
             parser = ElementTree.XMLParser(target=WhitespaceNormalizer())
+            # Remove BOM, or a Byte Order Mark.  http://en.wikipedia.org/wiki/Byte_order_mark
+            if pom_string.startswith("ï»¿"):
```

Comment:
SyntaxError: Non-ASCII character '\xc3' in file /home/travis/build/jiptool/jip/jip/maven.py on line 114, but no encoding declared; see http://python.org/dev/peps/pep-0263/ for details